omero/developers/whatsnew.txt

```diff
@@ -7,7 +7,9 @@ What's new for OMERO 5.1
   (currently slated to be 5.2). Most notably for development purposes, we are
   deprecating support for Java 1.6 and Python 2.6.
 - :doc:`Model/Units` introduces the new Units-types which are available
-  for use from all model objects.
+  for use from :doc:`Model Objects <Model/EveryObject>` (note that this
```

Comment:
Don't know if I'll be able to open a PR this week, but somewhere here we need something along the lines of "a number of objects formerly of numerical type are now unit quantities. Code working with these types will no longer compile. Queries of these types will now return different values." An example is pretty easy based on the additions to `Model/Units.txt`.